src/features/serde/mod.rs

```diff
@@ -87,6 +163,19 @@ impl serde_incl::ser::Error for crate::error::EncodeError {
     }
 }
 
+#[cfg(not(feature = "std"))]
+impl serde_incl::de::StdError for crate::error::EncodeError {}
```

Comment:
Wait, `StdError` exists when _not_ in `std`?

Reply:
https://github.com/serde-rs/serde/blob/55a7cedd737278a9d75a2efd038c6f38b8c38bd6/serde/src/ser/mod.rs#L118-L123
https://github.com/serde-rs/serde/blob/55a7cedd737278a9d75a2efd038c6f38b8c38bd6/serde/src/std_error.rs

They have an alias for a different error trait that's swapped out when `no_std`